__tests__/app.test.js

```diff
@@ -127,50 +127,163 @@ describe("API's", () => {
+        return request(app)
+          .post("/api/articles/1/comments")
+          .send({
+            username: "butter_bridge",
+            body: "Hello New Post",
+          })
+          .expect(201)
+          .then(({ body }) => {
+            const { comment } = body;
+            comment.created_at = new Date(comment.created_at);
+            expect(comment).toMatchObject(
+              expect.objectContaining({
+                comment_id: 19,
+                body: "Hello New Post",
+                article_id: 1,
+                author: "butter_bridge",
+                votes: 0,
+              })
+            );
+            expect(comment).toMatchObject({ created_at: expect.any(Date) });
+          });
+      });
+      test("POST 201: Should add a comment for an article and return the added comment even if the body an empty string", () => {
+        return request(app)
+          .post("/api/articles/1/comments")
+          .send({
+            username: "butter_bridge",
+            body: "",
+          })
+          .expect(201)
+          .then(({ body }) => {
+            const { comment } = body;
+            comment.created_at = new Date(comment.created_at);
+            expect(comment).toMatchObject(
+              expect.objectContaining({
+                comment_id: 19,
+                body: "",
+                article_id: 1,
+                author: "butter_bridge",
+                votes: 0,
+              })
+            );
+            expect(comment).toMatchObject({ created_at: expect.any(Date) });
+          });
+      });
+      test("POST 422: Should return an error if the article_id is a valid datatype but does not exsist", () => {
+        return request(app)
+          .post("/api/articles/14/comments")
+          .send({
+            username: "butter_bridge",
+            body: "Hello New Post",
+          })
+          .expect(422)
+          .then(({ body }) => {
+            const { msg } = body;
+            expect(msg).toBe("Not Found");
+          });
+      });
+      test("POST 400: Should return an error if the article_id is not valid datatype or not included", () => {
+        return request(app)
+          .post("/api/articles/invalid-id/comments")
+          .send({
+            username: "butter_bridge",
+            body: "Hello New Post",
+          })
+          .expect(400)
+          .then(({ body }) => {
+            const { msg } = body;
+            expect(msg).toBe("Bad Request");
+          });
+      });
+      test("POST 422: Should return an error if the username is a valid data type but does not exist", () => {
+        return request(app)
+          .post("/api/articles/1/comments")
+          .send({
+            username: "not_a_valid_username",
+            body: "Hello New Post",
+          })
+          .expect(422)
+          .then(({ body }) => {
+            const { msg } = body;
+            expect(msg).toBe("Not Found");
+          });
+      });
```

Comment:
I'd consider keeping in line with your previous tests such as line **76: GET 404: Responds with the appropriate error when passed the wrong article_id** I understand why you have chosen to use this status code however, try to keep consistency across your code and write this as a 404. (Same applies with the other 422 above 👀 )